go/consensus/cometbft/apps/roothash/state/state.go

```diff
@@ -303,6 +399,93 @@ func (s *MutableState) SetRuntimeState(ctx context.Context, state *roothash.Runt
 	if err := s.ms.Insert(ctx, ioRootKeyFmt.Encode(&state.Runtime.ID), ioRoot); err != nil {
```

Comment:
@kostko Can we remove this? Don't see anyone using it.

Reply:
AFAIK nothing is using it and now that there are historic roots we could remove it. I/O roots are useful for proving runtime transactions and events.

Reply:
Done in https://github.com/oasisprotocol/oasis-core/pull/5377.

Reply:
I was only referring to the IO root. We cannot simply remove the state root as the existing runtimes are using it for light client verification.